src/main/java/gyro/google/compute/InstanceFinder.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2019, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.compute;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.google.api.services.compute.Compute;
+import com.google.api.services.compute.model.Instance;
+import com.google.api.services.compute.model.InstanceList;
+import com.google.api.services.compute.model.Zone;
+import gyro.core.Type;
+import gyro.google.GoogleFinder;
+
+/**
+ * Query instance.
+ *
+ * Example
+ * -------
+ *
+ * .. code-block:: gyro
+ *
+ *    instance: $(external-query google::instance {zone: 'us-west1-a', filter: 'name = gyro-development'})
+ */
+@Type("instance")
```

Comment:
Prefix this with `compute-` and update the examples with the new type:
```suggestion
@Type("compute-instance")
```

Reply:
Done.